packages/blocks/src/_common/embed-block-helper/helper.ts

```diff
@@ -48,3 +54,78 @@ export function createEmbedBlock<
     view,
   };
 }
+
+// ========== Link Preview ==========
+
+const linkPreviewEndpoint = (() => {
```

Comment:
The worker endpoint here is a global singleton, which does not support multiple editors well. For instance, if two editors use different workers, I suggest managing the instances within the service.

Reply:
but still need to share instances in a single editor instance, for example, link preview is used in bookmark/youtube/github block. does blocksuite currently provide an editor-level config sharing mechanism?

Reply:
For me, it's acceptable that each block service maintains an instance. Currently, our inline style configurations also follow one per block.

<img width="643" alt="image" src="https://github.com/toeverything/blocksuite/assets/50035259/83ca62ae-b043-4dc4-ad72-8d8cc56ee2a8">

Reply:
Blocksuite is usually configured for block-level, focusing on blocks rather than the editor.

Reply:
fixed